apps/backend/local-stack/docker/voice/stt_server.py

```
@@ -0,0 +1,833 @@
+                    return
+
+                now = _now_ms()
+                if audio_start == 0:
+                    audio_start = now
+                bytes_received += len(payload)
+                # Sample-rate plausibility over the audio clock (AC-6): a
+                # stream that delivers far more (or far less) than 32000
+                # bytes per second is not 16 kHz mono 16-bit PCM. The floor
+                # lets a real-time client burst a few hundred ms before
+                # judging; the ±30% window tolerates network jitter while
+                # still rejecting e.g. 44.1k stereo (~5.5× the rate).
```

Ruff (0.16.1)

[warning] Comment contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF003)